### PR TITLE
Move ci script to Github Actions

### PR DESCRIPTION
**Pull request by vincenzobaz**
This PR introduces a Github Action workflow for the repositry

### CODE REVIEW

**Comment by vincenzobaz** [comment]
The UI looks like [this](https://github.com/vincenzobaz/docs.scala-lang/runs/1776302291?check_suite_focus=true)

**Comment by vincenzobaz** [comment]
I think it is appropriate to use the "Squash and merge" button if the review is positive

File: scripts/ci.sh [hunk 1]
```diff
@@ -1,21 +0,0 @@
...
-. /usr/local/rvm/scripts/rvm
-
-set -eux
-
-bundle install
-./scripts/run-mdoc.sh
-rm -r /tmp/mdoc-out/
-bundle exec jekyll build
-
-# Checking for docs.scala-lang/blob/master leads to a chicken and egg problem because of the edit links of new pages.
```
Comment (sjrd) [inline scripts/ci.sh:12]: We should preserve this comment in the new `build.yml`.

**Review by sjrd — CHANGES_REQUESTED** [review]
We should also remove `.drone.yml`, I believe.

**Review by sjrd — APPROVED** [review]

**Review by SethTisue — APPROVED** [review]

**Pull request merged**